.github/workflows/rtw-env/rtw-env.yml

```diff
@@ -0,0 +1,11 @@
+name: recipe_test_workflow
+
+channels:
+  - conda-forge
```

Comment:
```suggestion
  - conda-forge
  - nodefaults
```

Reply:
I'd add the restrictive `nodefaults` (no package off the problematic `main` channel, strict) if I was you, just in case

Reply:
I've added this in commit 41b0abb8b4096738d1b3a4b803fd933b1c0e5da1 . Thanks :slightly_smiling_face: